### ISSUE TITLE
Stuck

### ISSUE THREAD

**Issue by HyperChronos27** [SirYodah/NieR_Android_Theme#8]
So I open my phone, animations play as normal accompanied by all sounds and such, but there's a bit of an issue. The problem is that whenever I swipe between screens, nothing changes, I'm still on media, which is where I start out for some reason.

**Comment by Snek4Prez** [comment]
Yeah, same problem here, which really sucks since this theme looks sick

**Comment by ANMLxBrandon1** [comment]
By default the nova launcher only has one screen page, which causes you to only be able to view one screen. Long hold the screen and add two pages making the center your home screen.
